configuration_helper/models/config.py

```diff
@@ -0,0 +1,53 @@
+# © 2014 David BEAL Akretion
+# © 2016 Yannick Vaucher (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import re
+
+from odoo import api, fields, models
+
+
+class AbstractConfigSettings(models.AbstractModel):
+    _name = 'abstract.config.settings'
+    _description = 'Abstract configuration settings'
+    # prefix field name to differentiate fields in company with those in config
+    _prefix = 'setting_'
+    # this is the class name to import in your module
+    # (it should be ResCompany or res_company, depends of your code)
+    _companyObject = None
```

Comment:
Spotted with @p-tombez, as now the config settings are all in `res.config.settings`, this can't work as only the last `_inherit` would win. @p-tombez is working on this module.